### ISSUE TITLE
blocked invoices

### ISSUE THREAD

**Issue by saud-2** [lavaloon-eg/ksa_compliance#157]
I have three invoices are blocked by validation because total net amount are not equal to sum of line items taxable amounts ( diff is one hallah) . example :
![image](https://github.com/user-attachments/assets/5293220b-34fa-42ad-bf41-2fbc40cdaa02)

![image](https://github.com/user-attachments/assets/d777cf37-1ee0-4e21-8285-2f28f4732f5e)
 
rounding issue . is there any way to fix it.

tax are include it

Regards

**Comment by mhaggag** [comment]
We have reported this issue to frappe on [github](https://github.com/frappe/erpnext/issues/43143) as well as internally using a support ticket on frappe cloud and they said they'll fix it in an upcoming release. I don't know if they're tracking it on the issue linked above or another issue. I've asked for details and will get back to you once I have them.

**Comment by saud-2** [comment]
I changed the settings as per recommendation but the issue is still not resolved.

**Comment by mhaggag** [comment]
> I changed the settings as per recommendation but the issue is still not resolved.

Can you update to ERPNext 15.42.0 and check? That release includes a fix for the ERPNext bug mentioned above.

**Comment by saud-2** [comment]
The issue is resolved after updating.
Thank you and appreciate your effort.

**Comment by saud-2** [comment]
unfortunately, the issue is not solved completely. 
e.g:

![image](https://github.com/user-attachments/assets/25cfb959-1c39-40ec-9809-30310535b222)

![image](https://github.com/user-attachments/assets/33d04fc0-677c-459f-ad00-5a53a5882ec5)
